src/main/java/io/syndesis/qe/marketplace/openshift/OpenShiftService.java

```diff
@@ -38,6 +45,8 @@
     @Getter
     private OpenShift openShiftClientAsRegularUser;
 
+    private Map<String, List<PackageManifest>> catalog;
```

Comment:
Is there a reason to have this value cached? I'd say brings more confusion to the getChannel method as result of getCatalog looks like it's ignored.

Reply:
this cache avoid the API invocation every time , I suppose the catalog is not often updated. It is useful when you want install more than one operator. I added 'refreshCatalog' parameter in the getChannel method to handle the use of the cache as well

Reply:
cache removed, it will be used client side, if needed